### ISSUE TITLE
Extra screenshot on Fedora 35/Gnome

### ISSUE THREAD

**Issue by bognolo** [flameshot-org/flameshot#2112]
Hello, thank you for this amazing piece of software.
I'm on Fedora 35/Gnome and am happy you resolved the issue of Flameshot not being able to capture the screen after my recent upgrade to Fedora 35.
However, now triggering the screenshot (by clicking on the tray icon as usual) first takes a full screen screenshot (with shutter sound and everything), and shows the attached dialog, and only once permission to share is given Flameshot continues with the normal behaviour.
Can you please investigate the issue and restore normal functionality?

Thank you

![image](https://user-images.githubusercontent.com/5028387/143319944-108278f3-9193-453b-98f9-2b225024a12c.png)

**Comment by borgmanJeremy** [comment]
Nope, if you want the super details answer checkout the pinned thread.

Tldr: gnome changes the api to improve security and this is the only way to capture the screen. This is determined by gnome and not us. We are working with the portal devs to come up with a new system that only prompts you once but no one is actively working it.

**Comment by bognolo** [comment]
Thank you
